homeassistant/components/ecodevices/translations/en.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Device is already configured"
+    },
+    "error": {
+      "cannot_connect": "Failed to connect",
+      "invalid_auth": "Invalid authentication",
+      "unknown": "Unexpected error"
+    },
+    "step": {
+      "user": {
+        "title": "Parameters of Eco-Device",
+        "description": "Set connexion settings and select which input you use.",
+        "data": {
+          "name": "Name",
+          "host": "Host",
+          "port": "Port",
+          "scan_interval": "Seconds between updates",
+          "username": "Username",
+          "password": "Password",
+          "t1_enabled": "Teleinfo 1",
+          "t2_enabled": "Teleinfo 2",
+          "c1_enabled": "Meter 1",
+          "c2_enabled": "Meter 2"
+        }
+      },
+      "params": {
+        "title": "Parameters of Eco-Device entities",
+        "data": {
+          "t1_name": "Teleinfo 1 Name",
```

Comment:
Why can people set the name? This is something that they can override in the entity registry of HA as well.